### ISSUE TITLE
name too tech-bro?

### ISSUE THREAD

**Issue by davidwhogg** [davidwhogg/GaiaSprint#40]
@megbedell points out that the word "Sprint" might imply tech world things that we don't want to imply.

**Comment by adrn** [comment]
Ah the proverbial problem. Does @megbedell have other suggestions? I think we've been crushed by this problem (of finding a better name) before in trying to rename hack day / hack week, no?

**Comment by davidwhogg** [comment]
Yes, yes we have. The AAS Hack Day went to AAS Hack Together Day.

**Comment by megbedell** [comment]
Gaia Co-working Week? Gaia Co-Op? Gaia Jam Session? (jk... or am I)

Other people may disagree but in my opinion "hack day/week" is still better than "sprint" because the latter has Silicon Valley cult associations whereas "hacking" is a more widespread term these days.

**Comment by abonaca** [comment]
Just to provide another viewpoint -- I haven't realized until just now that there are non-running associations with the word "sprint".

**Comment by adrn** [comment]
Yea, those are all good issues to bring up. I'm definitely not opposed to renaming!

One of the things that makes this challenging is that people have conflicting opinions about what words are hostile. In trying to rename AAS Hack Day, we found that some people are strongly against the word "hack" because of other things it may imply to people unfamiliar with the more recent usage of the word.

**Comment by adrn** [comment]
I guess "strongly against" is not quite right: some people expressed that they found the word "hack" intimidating because they thought we were going to hack the CIA.

**Comment by wilmatrick** [comment]
Can we think of something along the line of "The Gaia Jump-in" (or whatever works in English...)? We keep the sports analogy, but it's also about coming together, having fun, immediately getting our hands dirty with the new data...

**Comment by abonaca** [comment]
Just out of curiosity, what would "The Gaia Jump-in" sound in German?

**Comment by wilmatrick** [comment]
Oh my, there is no German translation... I thought it might work in English, "jump-in" like "get-together"? No?

**Comment by megbedell** [comment]
I guess "kick-off" would be a similar sports analogy? Although as @davidwhogg pointed out there is a connotation of trying to finish projects/papers with "sprint," whereas "kick-off" or similar loses that sense of a finish line in sight.

**Comment by abonaca** [comment]
There is definitely a more distinct sense of urgency implied by "sprint", although I think "kick-off" might have ended up being a more accurate description of the past meetings :)

**Comment by wilmatrick** [comment]
Another idea: "Gaia Mining Week"? It's reminiscent of "data mining", and also about getting "gems" out of the dirty data, i.e. pretty little projects that can be done in one week? 

Other than that, I have talked to someone working in industry yesterday, and the way how they understand "sprint" in the context of the scrum technique is actually much closer to the concept of our Gaia Sprint than the sports analogy that I thought we had originally in mind. https://www.scrum.org/resources/blog/typical-sprint-play-play
So now I don't see the problem with using "sprint" in the tech world sense and like the name even better than before.

**Comment by megbedell** [comment]
Yeah, I agree that "sprint" is an accurate description in the tech-world sense, my objection is just that adopting that terminology builds associations with bro-ey tech organizations and that may be counter to increasing diversity at the events.

That said, it does seem like the vast majority of people won't make that association anyway, so I've personally come around to "sprint" as the best option.

**Comment by davidwhogg** [comment]
@abonaca says we should call it the **Gaia Stroll** !

**Comment by jradavenport** [comment]
As others point out, co-working, hack, and sprint all have Tech meaning that we may not want to embrace, and has plagued us for (re)branding the AAS end-of-week-event.

Off the cuff alternatives: **Gaia Retreat**, **Gaia-fest**, **Gaiathon** (or _Gaiathalon_ if we want to really ace the soundalike portmanteau), **Gaia Storm**, or **Gaia Create**

I don't know that changing Gaia Sprint now is key, but coming up with new names/terms to call this growing class of events that minimize tech/bro/etc ideation is a useful exercise!
